packages/types/src/api/web/role.ts

```diff
@@ -1,12 +1,14 @@
 import { Role, RoleUIMetadata } from "../../documents"
+import { PermissionLevel } from "../../sdk"
 
 export interface SaveRoleRequest {
   _id?: string
   _rev?: string
   name: string
-  inherits: string
+  inherits?: string | string[]
```

Comment:
Is this meant to be optional?

Reply:
This is correct, the `Role` has always allowed inherits to be empty - it could be worth refactoring this as I think inherits can always be set now, but its quite big and would probably do outside this RP.